controllers/teams.js

```diff
@@ -919,9 +920,23 @@ router.delete('/:teamId/members', async function(req, res, next) {
     res.sendStatus(200);
 });
 
+router.patch('/:teamId/invitation', async function(req, res, next) {
```

Comment:
why not move this to the server?

Reply:
It is only a proxy route. The logic is already handle from the api.